packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -260,8 +260,52 @@ class _RenderVectorGraphics extends RenderProxyBox {
     markNeedsPaint();
   }
 
+  @override
+  bool hitTestSelf(Offset position) => true;
+
+  @override
+  bool get sizedByParent => true;
+
+  @override
+  Size computeDryLayout(BoxConstraints constraints) {
+    return constraints.smallest;
+  }
+
+  @override
+  bool get isRepaintBoundary => true;
+
+  final Matrix4 transform = Matrix4.identity();
+
   @override
   void paint(PaintingContext context, ui.Offset offset) {
+    if (_scaleCanvasToViewBox(transform, size, pictureInfo.size)) {
+      context.canvas.transform(transform.storage);
+    }
     context.canvas.drawPicture(_pictureInfo.picture);
   }
 }
+
+bool _scaleCanvasToViewBox(
```

Comment:
This is more or less the same except instead of viewbox it just uses picturesize